pylint_django/checkers/migrations.py

```diff
@@ -0,0 +1,41 @@
+import astroid
+from pylint import interfaces
+from pylint.checkers import utils, BaseChecker
+from pylint_django.__pkginfo__ import BASE_ID
+
+def _is_migrations_module(node):
+    if not isinstance(node, astroid.Module):
+        return False
+
+    return 'migrations' in node.path[0] and not node.path[0].endswith('__init__.py')
```

Comment:
Note: this function is duplicate between here and db_performance.py so it would probably need to be moved to a common place. I think utils.py is a good place.